core/arch/arm/kernel/abort.c

```diff
@@ -568,3 +568,15 @@ void abort_handler(uint32_t abort_type, struct thread_abort_regs *regs)
 		break;
 	}
 }
+
+#ifdef CFG_CORE_WORKAROUND_ARM_NMFI
```

Comment:
This code should probably be put in `core/arch/arm/kernel/thread.c` instead.

Reply:
Ok. Lets see.